includes/utils/class-amp-dom-utils.php

```diff
@@ -75,6 +75,13 @@ public static function get_dom( $document ) {
 			$document
 		);
 
+		// Force all self-closing tags to have closing tags since DOMDocument isn't fully aware.
+		$document = preg_replace(
+			'#<(' . join( self::$self_closing_tags, '|' ) . ')[^>]*>(?!</\1>)#',
```

Comment:
No changes require: while both work, usually the glue string is the first parameter when using `join()` which is an alias of `implode()`.